portfolio/src/main/java/com/google/sps/servlets/AuthServlet.java

```diff
@@ -0,0 +1,55 @@
+// Copyright 2019 Google LLC
```

Comment:
It seems indentation is off in this whole file, please fix

Reply:
Resolved via chat: https://google.github.io/styleguide/javaguide.html#s4.2-block-indentation